### ISSUE TITLE
spiffs_page_object_ix_header._align is not properly zeroed

### ISSUE THREAD

**Issue by rojer** [pellepl/spiffs#171]
I am working with a flash controller that does not like it when application tried to set bits from 0 to 1.
naturally, it's not possible, and most controllers just ignore it, but not this one - it raises an error for such writes, and I found that SPIFFS is a bit sloppy in this regard.
In particular, the dummy _align member of [spiffs_page_object_ix_header](https://github.com/pellepl/spiffs/blob/f5e26c4e933189593a71c6b82cda381a7b21e41c/src/spiffs_nucleus.h#L501) ends up with having garbage data sometimes - in particular during operations in `spiffs_object_append`. it should be zeroed or left at 0xff, or set to something else - in any case, it needs to have stable value.

**Comment by pellepl** [comment]
Right, the _align member should be zeroed (or something). The transition 0->1 discussion we take in #173. :)
